src/server/Privilege.ts

```
@@ -32,33 +21,37 @@ export class PrivilegeChecker {
       return true;
     }
 
-    for (const groupName of allowedGroups) {
-      const groupRoles = this.cosmetics.role_group?.[groupName] || [];
+    for (const groupName of role_group) {
       if (
         roles !== undefined &&
-        roles.some((role) => groupRoles.includes(role))
+        roles.some((role) =>
+          this.cosmetics.role_groups[groupName].includes(role),
+        )
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)